worker/scripts/get-dep.sh

```diff
@@ -158,6 +158,15 @@ function get_fuzzer_corpora()
 	get_dep "${GIT_REPO}" "${GIT_TAG}" "${DEST}"
 }
 
+function get_win_getopt()
+{
+	GIT_REPO="git@github.com:alex85k/wingetopt.git"
+	GIT_TAG="master"
+	DEST="deps/getopt/getopt"
+
+	get_dep "${GIT_REPO}" "${GIT_TAG}" "${DEST}"
+}
+
 case "${DEP}" in
 	'-h')
 		echo "Usage:"
```

Comment:
Append `|win-getopt"` to the next echo usage line.

Reply:
yep. updated.